pipeline/pipeline.yml

```
@@ -37,90 +35,55 @@ groups:
   - terraform-destroy
 
 resource_types:
-
 - name: terraform
-  type: docker-image
+  type: registry-image
   source:
-    repository: ljfranklin/terraform-resource
-    tag: ((terraform_version))
+    repository: cycloid/terraform-resource
+    tag: "1.11.1"
 
 resources:
-
 - name: tfstate
   type: terraform
   icon: terraform
   source:
     env_name: ($ .environment $)
-($- if eq .use_case "aws-ec2" $)
-    backend_type: s3
-    backend_config:
-      bucket: ((terraform_storage_bucket_name))
-      key: ($ .project $)-($ .environment $).tfstate
-      workspace_key_prefix: ($ .project $)
-      region: ((aws_default_region))
-      access_key: ((aws_access_key))
-      secret_key: ((aws_secret_key))
-    vars:
-      access_key: ((aws_access_key))
-      secret_key: ((aws_secret_key))
-      aws_region: ((aws_default_region))
-      env: ($ .environment $)
-      project: ($ .project $)
-      customer: ($ .organization_canonical $)
-    env:
-      AWS_ACCESS_KEY_ID: ((aws_access_key))
-      AWS_DEFAULT_REGION: ((aws_default_region))
-      AWS_SECRET_ACCESS_KEY: ((aws_secret_key))
-($- else if eq .use_case "azure-linux" $)
-    backend_type: azurerm
+    backend_type: http
     backend_config:
-      container_name: ((terraform_storage_container_name))
-      key: ((terraform_storage_container_path))/((project))-((env)).tfstate
-      resource_group_name: ((terraform_resource_group_name))
-      storage_account_name: ((terraform_storage_account_name))
-      access_key: ((terraform_storage_access_key))
+      address: '($ .api_url $)/inventory?jwt=($ .inventory_jwt $)'
+      skip_cert_verification: true
     vars:
-      azure_location: ((azure_location))
-      azure_env: ((azure_env))
-      azure_client_id: ((azure_client_id))
-      azure_client_secret: ((azure_client_secret))
-      azure_subscription_id: ((azure_subscription_id))
-      azure_tenant_id: ((azure_tenant_id))
-      os_admin_password: ((os_admin_password))
       env: ($ .environment $)
```

```suggestion
      env: ($ .env $)
```

.env is the standard now